manual/src/installation_features.md

```diff
@@ -19,22 +19,23 @@ features = ["async-std-runtime"]
 ```
 
 ## Enabling the sync API
-The driver also provides a blocking sync API. To enable this, add the `"sync"` feature to your `Cargo.toml`:
+The driver also provides a blocking sync API. To enable this, add the `"sync"` or `"tokio-sync"` feature to your `Cargo.toml`:
 ```toml
 [dependencies.mongodb]
```

Comment:
this guidance can be simplified to omit `default-features = false` now if we use `tokio-sync`. We may still want to include more details about how to use async-std with sync too though. Also, make sure to update the README to reflect these changes too.

This just gave me the thought: should we add an `async-std-sync` feature and just remove mentions of `sync` from our docs? The old feature will continue to work, but this new one might be easier to understand.

In 3.0, I think we'd want to go back to just having a single `sync` feature flag and delegate the runtime choice to the `async-std-runtime` or `tokio-runtime` feature flags though.

Reply:
Updated the documentation here, in the README, and in `lib.rs`.

I agree on how things should work in 3.0.  For now, however, given that there'd still need to be the `default-features` distinction between `tokio-sync` and `async-std-sync`, I don't think adding the latter would be much of a gain in clarity or worth the churn.